src/System Application/Test/Azure Blob Services API/src/ABSBlobClientTest.Codeunit.al

```diff
@@ -787,6 +787,52 @@ codeunit 132920 "ABS Blob Client Test"
         VerifyContainerContentType(ABSContainerContent, CopyStr(ABSTestLibrary.GetSampleResponseFileName(), 1, MaxStrLen(ABSContainerContent.Name)), Enum::"ABS Blob Resource Type"::File);
     end;
 
+    [Test] // Failing if ran against azurite, as BLOB MetaData are not supported there
+    procedure GetBlockBlobMetaDataTest()
```

Comment:
Given this test is failing in the CI pipeline, you must disable it here: https://github.com/microsoft/BCApps/blob/main/src/System%20Application/Test/DisabledTests/ABSBlobClientTest.json

Reply:
done